connector/spanmetricsconnector/config.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 const (
-	delta      = "AGGREGATION_TEMPORALITY_DELTA"
-	cumulative = "AGGREGATION_TEMPORALITY_CUMULATIVE"
+	delta                           = "AGGREGATION_TEMPORALITY_DELTA"
+	cumulative                      = "AGGREGATION_TEMPORALITY_CUMULATIVE"
+	dynamicExemplarAttributeKeyName = "tail_sampling.sampled"
```

Comment:
I think the folks from SIG Sampling should have a say on this feature as a whole, but especially on this attribute, as it has the potential for becoming a semconv to be used by other components.

cc @jmacd, @kentquirk

Reply:
This attribute is configurable so I don't see a reason why to introduce it as a convention.

Reply:
> This attribute is configurable so I don't see a reason why to introduce it as a convention.

It's more if we introduce the notion of a sampling key, does it align with what other groups have envisioned or can it be used across more projects.